### PR TITLE
add root layouts for widgets, XM1-35428

### PR DESCRIPTION
**Pull request by mak7iliano**

### CODE REVIEW

**Review by zverbeta — APPROVED** [review]

**Review by andrewkikot — APPROVED** [review]

**Review by denisogolubev — APPROVED** [review]

File: src/app/layouts/components/layout-wrapper/layout-wrapper.component.ts [hunk 1]
```diff
@@ -39,6 +48,12 @@ export class LayoutWrapperComponent implements OnInit, AfterViewInit, OnDestroy
         this.observeIsMaterial3Menu();
         this.observeIsMobileScreen();
         this.menuCategories$ = this.menuService.menuCategories;
+
+        this.uiConfigService.config$().pipe(
+            takeUntilOnDestroy(this),
+        ).subscribe((config) => {
+            this.rootLayout = config.root?.layout || null;
```
Comment (sergeysenja1992) [inline src/app/layouts/components/layout-wrapper/layout-wrapper.component.ts:55]: Can we have conflict with existing root layout configuration?

Comment (mak7iliano) [reply src/app/layouts/components/layout-wrapper/layout-wrapper.component.ts:55]: root - it is key

Comment (mak7iliano) [reply src/app/layouts/components/layout-wrapper/layout-wrapper.component.ts:55]: Знімок екрана 2024-09-24 о 10 43 41

**Review by sergeysenja1992 — APPROVED** [review]

File: src/app/layouts/components/layout-wrapper/layout-wrapper.component.ts (hunk 1 above)
Comment (sergeysenja1992) [reply src/app/layouts/components/layout-wrapper/layout-wrapper.component.ts:55]: but we already have configurations like this
image

Comment (sergeysenja1992) [reply src/app/layouts/components/layout-wrapper/layout-wrapper.component.ts:55]: better to use another more describable name

Comment (mak7iliano) [reply src/app/layouts/components/layout-wrapper/layout-wrapper.component.ts:55]: may be "base" or "body"

Comment (denisogolubev) [reply src/app/layouts/components/layout-wrapper/layout-wrapper.component.ts:55]: It can be `main`, because it used in `main.component`

**Pull request merged**